src/apps/rDSN.monitor/app_package/static/js/view.js

```diff
@@ -0,0 +1,270 @@
+function getParameterByName(name) {
+    name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
```

Comment:
the file name 'view.js' is too general, please be specific about what it is about.